internal/store/files/recog_place_store.go

```diff
@@ -63,6 +63,18 @@ var (
 		"state",
 		"states",
 	}
+	// Map of entity types to their human string.
```

Comment:
TODO:  move this to a file?

Another thought:  should we just put the human strings directly in "CommonWordReconNameTo..." map?  Is there a reason the value needs to be a type?

Reply:
I don't think there's actually a reason the value needs to be a type, updated to just put the human strings directly in the map